.stylelintrc

```diff
@@ -0,0 +1,73 @@
+{
+  "plugins": [
+    "stylelint-order",
+    "stylelint-scss",
+    "stylelint-selector-bem-pattern"
+  ],
+  "extends": [
+    "stylelint-config-standard",
+    "stylelint-config-property-sort-order-smacss"
+  ],
+  "rules": {
+    "comment-empty-line-before": [
+      "always",
+      {
+        "except": ["first-nested"],
+        "ignore": ["after-comment", "stylelint-commands"]
+      }
+    ],
+    "declaration-empty-line-before": [
+      "always",
+      {
+        "except": ["first-nested"],
+        "ignore": ["after-declaration"]
+      }
+    ],
+
+    "order/order": [
```

Comment:
Are these settings trying to mimic our current sorting configuration? Seems like a lot of configuration...

Reply:
So these are just setting the order of different elements of a block. So for example (line spaced for clarity)
```css
.selector {
  --custom: 'property';

  $sass: 'variable';

  @extend %placeholder-class;

  @include mixin();

  display: block;
  content: "smacss order";

  @supports (display: grid) {
    display: grid;
  }

  @include content-mixin() {
    max-width: 320px;
  }

  &:hover {}

  &::after {}

  .nested-selector {}
}
```

This sort of order has always made the most sense to me. If you don't think it's worth enforcing I can drop the rules.

Reply:
Dropped 'em anyway after I opened a PR on the main config to include them. 😉